Pls, rebase I have fixed the warning:
```
n function 'ul_getuserpw_str',
    inlined from 'main' at login-utils/chfn.c:452:12:
lib/pwdutils.c:227:27: error: 'num' may be used uninitialized [-Werror=maybe-uninitialized]
  227 |         if (rc == -ERANGE || num > MAX_OF_UINT_TYPE(uid_t)) {
      |                           ^
lib/pwdutils.c: In function 'main':
lib/pwdutils.c:220:18: note: 'num' was declared here
  220 |         uint64_t num;
      |                  ^
lto1: all warnings being treated as errors
```

--- a/sys-utils/Makemodule.am
+++ b/sys-utils/Makemodule.am
@@ -599,4 +599,7 @@ if HAVE_LINUX_LANDLOCK_H
 setpriv_SOURCES += sys-utils/setpriv-landlock.c
 endif
 setpriv_LDADD = $(LDADD) -lcap-ng libcommon.la
+if HAVE_ECONF
+setpriv_LDADD += $(ECONF_LIBS)
 endif
+endif # BUILD_SETPRIV

